root/types.js

```diff
@@ -690,6 +690,11 @@ declare type LinkTypeT = {
   +cardinality1: number,
   +children?: $ReadOnlyArray<LinkTypeT>,
   +deprecated: boolean,
+  +documentation?: string,
```

Comment:
I think this should be
```suggestion
  +documentation: string | null,
```
since `$json->{documentation} = $self->documentation;` means the property will always be present, though it may be serialized as `null` if `$self->documentation` is `undef` on the Perl side.